pom.xml

```diff
@@ -110,6 +110,7 @@ under the License.
         <surefire.module.config/>
 
         <derby.version>10.15.2.0</derby.version>
+        <jetbrains.annotations.version>24.1.0</jetbrains.annotations.version>
```

Comment:
Required by `KubernetesAutoScalerStateStore` https://github.com/apache/flink-kubernetes-operator/blob/968a578515d6269bbd5637594a7a342d74c1cd5c/flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/autoscaler/state/KubernetesAutoScalerStateStore.java#L205-L212 

though looking again that looks like a bug and it should use `@nonnull` instead.

Reply:
@gyfora @csviri Do you have any preferences as to how to resolve this? Should the annotation be updated or add the extra dependency?

Reply:
I've removed `@NotNull` from `KubernetesAutoScalerStateStore` as `@NonNull` is the default.